### PR TITLE
Rework day 3 schedule

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Reworking tomorrow's schedule:

- Starting with the RNA-seq slides
- Marking the NB cell line tximeta exercise as something we'll do if time allows
- Moving the EDA exercise to the consultation time at the end.

It did not seem worthwhile to predict how long the slides + the first notebook would take and shift the schedule. It seemed better to see how the slot right before lunch goes and adjust in the moment.

I also fixed a few minor things on the bulk slides this morning, so it seemed worthwhile to update the PDF.

### CODE REVIEW

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -56,18 +55,18 @@ Minneapolis, MN 55414 in Room 105.
 |-------------|------------------------------------------------|
 | **Day 3**   | **2024-08-21 <br> Bulk RNA-Seq**  |
 | 8:30 AM     | Breakfast and coffee available |
-| 9:00 AM     | [RNA-Seq quantification (trim, filter, quant)](../completed-notebooks/RNA-seq/01-qc_trim_quant.nb.html) |
+| 9:00 AM     | Introduction to bulk RNA-Seq <br> [Introduction to bulk RNA-seq slides (PDF)](../slides/2024-08-20_intro-to-bulk-RNAseq.pdf) <br> [RNA-Seq quantification (trim, filter, quant)](../completed-notebooks/RNA-seq/01-qc_trim_quant.nb.html) |
```
Comment (sjspielman) [inline workshop/SCHEDULE.md:58]: You would both have a slightly better sense than I would if this is enough time for both, so I'll let @jashapiro weigh in!

File: slides/2024-08-20_intro-to-bulk-RNAseq.pdf
Comment (sjspielman) [inline slides/2024-08-20_intro-to-bulk-RNAseq.pdf:1]: just noticed the "what you'll learn to do" in this module has black font (vs our dark gray font). up to you if this bothers you!

**Review by sjspielman — APPROVED** [review]
👍

File: workshop/SCHEDULE.md (hunk 1 above)
Comment (sjspielman) [reply workshop/SCHEDULE.md:58]: (noting that your opening comment does note that not necessarily worth estimating this, which is good w/ me!)

**Review by jashapiro — APPROVED** [review]

**Pull request merged**